runner/operator/access/v1_0_0/legacy/input_template.json.jinja2

```diff
@@ -31,15 +31,15 @@
     "fx_path": "/opt/common/CentOS_6/picard/picard-tools-1.96/FixMateInformation.jar",
     "bwa_path": "/opt/common/CentOS_6/bwa/bwa-0.7.5a/bwa",
     "abra_path": "/opt/common/CentOS_6/abra2/abra2-2.17/abra2-2.17.jar",
-    "arrg_path": "/home/johnsoni/vendor_tools/AddOrReplaceReadGroups-1.96.jar",
+    "arrg_path": "juno:///juno/work/access/production/resources/tools/picard/versions/v1.96/picard-tools-1.96/AddOrReplaceReadGroups.jar",
     "gatk_path": "/opt/common/CentOS_6/gatk/GenomeAnalysisTK-3.3-0/GenomeAnalysisTK.jar",
-    "waltz_path": "/home/johnsoni/vendor_tools/Waltz-2.0.jar",
-    "picard_path": "/home/johnsoni/vendor_tools/picard-2.8.1.jar",
+    "waltz_path": "juno:///juno/work/access/production/resources/tools/waltz/current/Waltz-3.2.0.jar",
+    "picard_path": "juno:///juno/work/access/production/resources/tools/picard/versions/v2.8.1/picard-2.8.1.jar",
```

Comment:
my apologies @buehlere  can you make use of this path `juno:///juno/work/access/production/resources/tools/picard/versions/v2.8.1/picard.jar`